tasks/cephfs/fuse_mount.py

```diff
@@ -100,11 +100,16 @@ def list_connections():
         self.fuse_daemon = proc
 
         # Wait for the connection reference to appear in /sys
+        mount_wait = self.client_config.get('mount_wait', 0)
+        if mount_wait > 0:
+            log.info("Fuse mount waits {0} seconds before checking /sys/")
+            time.sleep(mount_wait)            
```

Comment:
I did it this way because @gregsfortytwo suggested "add a wait time before the mount attempt to let the cluster get set up.". Did I get this wrong ? I'm not opposed to a 180s timeout, just following suggestions.